dali/python/nvidia/dali/pipeline.py

```diff
@@ -393,6 +393,8 @@ def _prepare_graph(self, define_graph = None):
             if isinstance(outputs[i], types.ScalarConstant):
                 import nvidia.dali.ops
                 outputs[i] = nvidia.dali.ops._instantiate_constant_node("cpu", outputs[i])
+            elif not isinstance(outputs[i], DataNode):
+                outputs[i] = types.Constant(outputs[i], device="cpu")
```

Comment:
Can we support cupy and GPU constants as well?

Reply:
Not at present.